RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -6457,7 +6457,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				RUN_DOCUMENTATION_COMPILER = YES;
 				SDKROOT = "";
-				SKIP_INSTALL = YES;
+				SKIP_INSTALL = NO;
```

Comment:
not strictly needed to have these set correctly since the fastlane lane does set them, but it's also a good idea to have them so that if we want to archive manually it works correctly. 

`SKIP_INSTALL` should be `NO` for frameworks, and `BUILD_LIBRARY_FOR_DISTRIBUTION` should be `YES`

Reply:
if you want to sanity check how these are set, sadly this is a terrible way to try to view it but you can go into xcode and look for the settings under build settings, and check them in the targets, i.e.: 

<img width="1589" alt="image" src="https://github.com/user-attachments/assets/793c678c-a55f-4021-8103-269cc49ed1cf">
<img width="1582" alt="image" src="https://github.com/user-attachments/assets/e343d408-11d7-4d37-929c-79b9872f5a75">